CASC/PortfolioMode.cpp

```diff
@@ -95,32 +95,30 @@ bool PortfolioMode::perform(Problem* problem)
   }
 
   if (outputAllowed()) {
-    env.beginOutput();
     if (resValue) {
-      addCommentSignForSZS(env.out());
-      env.out()<<"Success in time "<<Timer::msToSecondsString(env.timer->elapsedMilliseconds())<<endl;
+      addCommentSignForSZS(std::cout);
+      std::cout<<"Success in time "<<Timer::msToSecondsString(env.timer->elapsedMilliseconds())<<endl;
     }
```

Comment:
What do you think about `std::cout` vs just `cout` (and having the right namespace/using directive always active)? This is minor, but since we are already touching XXX files and it's just about a simple text replacing? 

I personally prefer the lazy (shorter) version.

And here it is particularly strange, as the fully qualified use is followed up by a short `endl`.

(Moreover, we could discuss whether `addCommentSignForSZS` shouldn't simply hardwire `std::cout` and be parameterless.)

Reply:
Could be either, matter of taste etc. Two mild points against unqualified `cout`:
1. This suggests that `using namespace std;` is somewhere floating around. This is true in Vampire but it shouldn't be. `using std::cout;` is OK but I think I'd prefer it scoped relatively close to use.
2. Generally I'd explicitly qualify `std::` so that when I actually want ADL magic it stands out.

But I'm not fussy, either works for me.

`addCommentSignForSZS` - absolutely. Although that's also used inconsistently so maybe that could/should go too.

Reply:
OK, let's prefer the fully qualified then.

With `addCommentSignForSZS` there seems to be at least one place in the code where we don't want to use `cout`, so let's keep it as is for now at least...